packages/presentation/lib/screen/movie_details/details_screen.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:domain/model/movie.dart';
+import 'package:flutter/material.dart';
+import 'package:presentation/base/bloc_data.dart';
+import 'package:presentation/base/bloc_screen.dart';
+import 'package:presentation/config/dimens/dimens.dart';
+import 'package:presentation/navigation/base_arguments.dart';
+import 'package:presentation/navigation/base_page.dart';
+import 'package:presentation/screen/movie_details/details_bloc.dart';
+import 'package:presentation/screen/movie_details/details_data.dart';
+import 'package:presentation/screen/movie_details/widgets/details_background_image.dart';
+import 'package:presentation/screen/movie_details/widgets/details_body.dart';
+import 'package:presentation/screen/movie_details/widgets/details_movie_image.dart';
+import 'package:presentation/screen/movie_details/widgets/details_shimmer.dart';
+import 'package:presentation/screen/movie_details/widgets/details_tab_bar.dart';
+
+class DetailsScreenArguments extends BaseArguments {
```

Comment:
what is the "movie_details_screen.dart" file for?

Reply:
forgot to delete it